packages/svelte/src/compiler/utils/ast.js

```diff
@@ -108,11 +108,8 @@ export function extract_all_identifiers_from_expression(expr) {
 		expr,
 		{},
 		{
-			Identifier(node, { path }) {
-				const parent = path.at(-1);
-				if (parent?.type !== 'MemberExpression' || parent.property !== node || parent.computed) {
-					nodes.push(node);
-				}
+			Identifier(node) {
+				nodes.push(node);
```

Comment:
This will fail other parts of the code. And this would probably also yield a false-positive match for `bind:group={group.a}` and `bind:group={group[a]}`. We need to find a different way to compare these, possibly by exact string match. @tanhauhau how did this work exactly in Svelte 4?

Reply:
Just want to point out another possibly tricky case, in `packages\svelte\tests\runtime-legacy\samples\binding-input-group-each-8\main.svelte`, where the group binding is using the key of a higher each loop, not the one it's in:

```svelte
{#each keys as key (key)}
  <h2>{key}</h2>
  <ul>
  {#each values as value (value)}
    <li>
      <label>
        <input type="checkbox" name={key} {value} bind:group={object[key]} /> {value}
      </label>
    </li>
  {/each}
  </ul>
{/each}
```

Reply:
My spontaneous thought is to use the bind:group expression itself as a key to the binding group object. What would be the problem with that?